File: URI/Haskell/1010.hs
import Text.Printf

main = do
    l1 <- getLine
    l2 <- getLine

    let lista1 = words l1
        lista2 = words l2
    printf "VALOR A PAGAR: R$ %.2f\n" $ total lista1 lista2

total :: [String] -> [String] -> Double 
total l1 l2 =
    let n1 = l1 !! 1
        n2 = l2 !! 1
        v1 = l1 !! 2
        v2 = l2 !! 2
    in read n1 * read v1 + read n2 * read v2 

File: URI/Java/src/URI/_1010.java
package URI;

import java.math.RoundingMode;
import java.text.DecimalFormat;
import java.util.Scanner;

public class _1010 {
    
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        DecimalFormat df = new DecimalFormat("0.00");
        df.setRoundingMode(RoundingMode.HALF_DOWN);
        
        sc.nextInt();
        int qtd = sc.nextInt();
        double valor = sc.nextDouble();
        double total = qtd * valor;
        
        sc.nextInt();
        qtd = sc.nextInt();
        valor = sc.nextDouble();
        total += qtd * valor;
        
        System.out.println("VALOR A PAGAR: R$ " + df.format(total));
    }
}
